src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -114,6 +114,10 @@ export const LandscapePageWrapper = styled.div`
 		width: 4.75rem;
 		height: 4.75rem;
 	}
+	.meshtitle-img-1 .vulcan-img.vulcan-img{
```

Comment:
why is `.vulcan-img` classname used twice? I dont think it will work 🤔 is it a mistake @rghdrizzle ?

Reply:
It was a mistake , updated it now.